### PR TITLE
Tutorials refactoring

### PR DESCRIPTION
**Pull request by karesti**
Depends on https://github.com/infinispan/infinispan-simple-tutorials/pull/132

### CODE REVIEW

File: _includes/tutorials-examples-band.html [hunk 1]
```diff
@@ -1,104 +1,11 @@
 <div class="component-slim">
   <div class="grid-wrapper">
     <div class="width-12-12">
-      <h3>Examples</h3>
-      <p class="margin-tb-0">Very simple, straight-to-the-point, code examples of specific Infinispan features and integrations. All of these are part of the <a href="https://github.com/infinispan/infinispan-simple-tutorials/" target="_blank" >Infinispan Simple Tutorials GitHub project</a>.</p>
+      <h3>Infinispan Simple Tutorials</h3>
```
[the remark this answers is not in the document]
Comment (oraNod) [reply _includes/tutorials-examples-band.html:4]: Not sure if this change is great or not but "Simple" just doesn't seem right to me tbh. It implies trivial or basic, which isn't always the case. I think the original intention was to make those examples lighter and easier to consume than some of the extended tutorials.

File: _data/ispn.yml [hunk 2]
```diff
@@ -1173,3 +1173,12 @@ sb_starter:
...
     title: docs/infinispan-spring-boot/9.4.x/spring_boot_starter.html
+
+# -----------------------------
+# Simple tutorials docs
+# -----------------------------
+
+simple_tutorials:
+  main:
+    zip-url: https://github.com/infinispan/infinispan-simple-tutorials/archive/main.zip
+    docs: /tutorials/simple/infinispan-simple-tutorials.html
```
Comment (oraNod) [inline _data/ispn.yml:1184]:
```suggestion
    docs: /tutorials/simple/simple_tutorials.html
```

File: _includes/tutorials-examples-band.html [hunk 3]
```diff
@@ -1,104 +1,11 @@
...
-      <p class="margin-tb-0">Using Infinispan as Hibernate Cache provider in a single-node standalone application.</p>
-    </div>
-    <div class="width-3-12 width-12-12-m margin-tb-sm">
-      <img src="{{site.baseurl}}/assets/images/tutorials/icon-example.png"> <a class="alt-type-b" href="{{site.baseurl}}/tutorials/simple/hibernate-cache-spring-local">Hibernate Cache Spring</a>
-      <p class="margin-tb-0">Using Infinispan as Hibernate Cache provider in a single-node Spring application.</p>
-    </div>
-    <div class="width-3-12 width-12-12-m margin-tb-sm">
-      <img src="{{site.baseurl}}/assets/images/tutorials/icon-example.png"> <a class="alt-type-b" href="{{site.baseurl}}/tutorials/simple/hibernate-cache-wildfly-local">Hibernate Cache WildFly</a>
-      <p class="margin-tb-0">Using Infinispan as Hibernate Cache provider in a single-node WildFly application.</p>
+      <img src="{{site.baseurl}}/assets/images/tutorials/icon-example.png"> <a class="alt-type-b" href="{{site.baseurl}}/infinispan-simple-tutorials">Infinispan Simple tutorials</a>
```
Comment (oraNod) [inline _includes/tutorials-examples-band.html:8]:
```suggestion
      <img src="{{site.baseurl}}/assets/images/tutorials/icon-example.png"> <a class="alt-type-b" href="{{site.data.ispn.simple_tutorials.main.docs}}">Infinispan Code Tutorials</a>
```

**Review by oraNod — CHANGES_REQUESTED** [review]

File: Gemfile.lock [hunk 4]
```diff
@@ -98,7 +97,6 @@ DEPENDENCIES
   minima (~> 2.0)
   nokogiri
   optimist
-  webrick
```
[the remark this answers is not in the document]
Comment (karesti) [reply Gemfile.lock]: oki

**Review by oraNod — APPROVED** [review]
LGTM

**Comment by oraNod** [comment]
thanks @karesti

**Pull request merged**